chainercv/datasets/transform_dataset.py

```diff
@@ -38,6 +38,9 @@ def __init__(self, dataset, transform):
         self._dataset = dataset
         self._transform = transform
 
+    def __getattr__(self, key):
+        return getattr(self._dataset, key)
+
```

Comment:
@yuyu2172 Hmm. OK. I reverted the change for `transform_dataset.py`. Please note that output image is now without labels.

![detection_idx 7_iter 0](https://cloud.githubusercontent.com/assets/1901008/23847899/eaf45702-0817-11e7-9c3d-6496e4f5c115.jpg)